storey/redis_driver.py

```diff
@@ -526,25 +547,35 @@ async def _load_aggregates_by_key(self, container, table_path, key):
         return aggregations_to_return, additional_data_to_return
 
     async def _fetch_state_by_key(self, aggr_item, container, table_path, key):
-        key = str(key)
+        redis_key_prefix = self._make_key(container, table_path, key)
+        aggregations = {}
         # Aggregation Redis keys start with the Redis key prefix for this Storey container, table
         # path, and "key," followed by ":aggr_"
-        aggregations = {}
-
         redis_key_prefix = self._make_key(container, table_path, key)
-        aggr_key_prefix = f"{redis_key_prefix}:{RedisDriver.AGGREGATION_ATTRIBUTE_PREFIX}"
-        all_aggr_keys = self.redis.hkeys(aggr_key_prefix)
-        for aggr_key in all_aggr_keys:
-            aggr_key = RedisDriver.convert_to_str(aggr_key)
-            value = await RedisDriver.asyncify(self.redis.hget)(aggr_key_prefix, aggr_key)
+        redis_key = self._static_data_key(redis_key_prefix)
+        try:
+            cursor = 0
+            values = {}
+            while True:
+                cursor, v = await RedisDriver.asyncify(self.redis.hscan)(
+                    redis_key, cursor, match=f"{self.AGGREGATION_ATTRIBUTE_PREFIX}*"
```

Comment:
Maybe we can parallelize these calls? i.e. with `asyncio.create_task` and `asyncio.gather`.